tests/FixerConfiguration/FixerConfigurationResolverRootlessTest.php

```diff
@@ -44,6 +44,6 @@ public function testResolveWithMappedRoot()
 
         static::assertSame($options, $configuration->getOptions());
 
-        $configuration->resolve(['baz', 'qux']);
+        $configuration->resolve(['baz' => 'qux']);
```

Comment:
wasn't goal of this test to have non-assoc array passed?

Reply:
Apparently not - test is still passing. According to expected deprecation message it tests config for creating resolver, not resolving.